### PR TITLE
feat: browser channel

### PR DESCRIPTION
**Pull request by kumaraditya303**
- [x] Create `browser_channel` fixture
- [x] Add tests for `browser_channel` fixture
- [x] Add docs

### CODE REVIEW

**Comment by kumaraditya303** [comment]
@mxschmitt Implementation is ready for review, after that we can consider adding docs

**Comment by mxschmitt** [comment]
@kumaraditya303 looks good to me, after moving to the normal Playwright stable release we can land this I think.

**Comment by kumaraditya303** [comment]
@mxschmitt I have added docs for `--browser-channel`, now you can consider merging this PR. :rocket:

File: README.md [hunk 1]
```diff
@@ -106,6 +106,18 @@ def test_visit_example(page):
     # ...
 ```
 
+### Run on chromium browser channel
+Start Pytest with `--browser-channel` argument.
```
Comment (mxschmitt) [inline README.md:110]:
```suggestion
### Run with a custom browser channel like Google Chrome or Microsoft Edge
```

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**